packages/coinstac-ui/app/render/log.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>COINSTAC Log</title>
+    <style>
+      * {
+        box-sizing: border-box;
+      }
+      html {
+        background: #333;
+        color: #fff;
+        font-family: 'Helvetica Neue', sans-serif;
+        height: 100%;
+        line-height: 1.5;
+      }
+      body {
+        height: 100%;
+        margin: 0;
+      }
+      h1 {
+        background: #555;
+        box-shadow: 0 .25rem .5rem rgba(0, 0, 0, .4);
+        color: #ccc;
+        font-size: 100%;
+        left: 0;
+        line-height: 3rem;
+        margin: 0;
+        padding: 0 1rem;
+        position: fixed;
+        top: 0;
+        width: 100%;
+      }
+      pre {
+        font-family: Menlo, Consolas, 'Courier New', Courier, monospace;
+        font-size: 14px;
+        height: 100%;
+        line-height: inherit;
+        margin: 0;
+        overflow: scroll;
+        padding: 4rem 1rem 1rem;
+      }
+      code {
+        font-size: 100%;
+        line-height: inherit;
+      }
+      .error {
+        color: red;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>COINSTAC Log</h1>
+
+    <pre id="output"></pre>
+
+    <script src="log.js"></script>
```

Comment:
Some virtue in divorcing the logger from the render UI build process, I think. We can always add it as a new entry in our Webpack bundles, but it’s simple enough to stand on its own for now.